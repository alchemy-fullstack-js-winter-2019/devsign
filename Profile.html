<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta http-equiv="X-UA-Compatible" content="ie=edge">
  <title>Twitter Clone</title>
</head>
<body>
  <header>
      <figure>
          <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAPAAAADSCAMAAABD772dAAABUFBMVEX///8AAADMzMzVWjHwmyS0MhjvWSjtaCbMSirNzc3JycnU1NTTVDLynyPQ0NCvLhf5+fns7Ozl5eXp6el0dHTY2Njg4OCEhIS9vb2mpqacnJxtbW1XV1f09PRSUlLvVShGRkaNjY3vkwC3t7dAQEAdHR2YmJh8fHzIMwDsVwAxMTEODg44ODhkZGQkJCRMTEzuSQAWFhbgm47kqJ3LRCH0rpTah3f98u6xIwDsYRXKQBnUUyXrwLn3xbPvUx335eL0qIzilH/xoTj0u3nce2DYZkPSSAn2uaP1n4jubS/62MvvzcfynXy6RTGyKQbwZzvxi2Lye1jRjoXxlG/5z7/v2dXKeW7otazBX1DudT7OXEXYoZmuCADTalXvf07yqU72xZD41rH75c387NzwfibxpUTZbk7zsmT1wIX52rnDZ1rEFgDwjSXvdibfhm3vdgDIWRzSAAAPdUlEQVR4nOWd+WPbthXHdbixSTaKLEuyZFtOGMmWbfmIczmx0yRtWmc9lrVLsq5d2/Rcd7Td///bCAIkARAPAElQMJ3vL7YlmcCHAN57eAChWs2eRuNti6XPW53hpF5fsV2Leam/fVgP5Nuux3zU82d1LNs1mYd6q/VYY9uVKV1J2yLt265Oyepsb9UZ9WzXqEyNhnt1Tuu261Sixps8baCR7VqVpVZXQFuvD23Xqxwtbx8LcQMfHGhtPB54vVHbdi2NaSDqyiLtHfhDt+phV2dVDcpzd4etqjb34EZmXKLD6bhybb3s56WN1B1XyIg3NorihtrfrkZkMtxSs2hr1bONo9BydkOlkt+3DQWrd2AcF2l/eDEttzspBTdU9+I18/iwPFykycA2IaNhubShZhcn+l6bA26oNdukobbnhXsxkOfWupHsdux5jF1eM3vmq3NkgTfQxJKTMh9VaWtqAXdgDxdp3jntkZkZUQEdLc+Td2wbF2l+9nqkm6oqWTfmlCSwPHppzWUki9PMllT+8sVKybOirDou2SdfoO4cqdRubTHWgNUtj9dSKKnSpKQU0MpMXbYllTKQW7apZHLN816I4AqW8bBr7hP9rDK8BepCmmdWRqeMFyq6gnRgjnfdNouejBGXs4JSgjbeMl5DxBXpz1gGevXUNkM2FQ6sfdsEWbVajNdGpr2gCq3GuLZrn0cFVib6tuueT7k3w4xs1zyv8mYzUzt+q6LDfLyVcsCscrnjChroRDlMdc92nYsp8462tu0aF1XWxJ711cGiOsrGW+kBjJVpGHds19aEsuRuK+uBaW3p8/q262pG2hOniobQaekG1ZeiQyNpduoLn3PXl1Z2ftl2LU1K5/mYyocctDTCj0omOWCp0x8XdxE4n1S8l8hiYSlOzqj8JCkteb6nAsuiWSVdRr1ULimSzDVVOI0FS5LgWrFdt3IEzxMvZQNLmviSNnC93gGAK7Yyqi9gDbWyKytqiX2xb7ta5Uk8TbRdqzIl4r3gewuLSbQzMXti53CyEWhzDzqMxbz2bqAC97PvyhesJ2ZaS9pb98euF8sZbx+YPOtAoNkBXWLLHfoHmVoovdakvbvweH0YlNxynEYsx2l53mBaGvPxdMyX2ApKdNf0t5ClnwbR+79ZNyy6IVAAPcx9PItMkyFcojfUjQ75xTUtk7U5DLqvqGiiANl4incvwJUW6a1pPZnAmy2NtukOpEWT8v2CgJxWdcocaDTzhOVVT4SnrrRxY3kDg418OPB0ynQ8V22C2Gmxanm0q4mLSm8ZS/RuiseuGFnVyuzyqbxVjgbauGHhhqaZ69lKVZxoxbhi+cRwLUvBSD0jO+jXe9lKdTx5yrWj2aM3GkqzkSY2sMd6Q2v4Mmq5sgd96T4t6QzbvYzNG8rbL8q7n7VbITkyJ0HZaXgmPBtD99nFgkouvGLjQrzygr0xvHSSzIrBZ0b3Rd0Zleb1+h2kfq8lLr1VcFvMmrhgtxUX7DXEBbtg50qeQ4VszGa6W7lur8PlD9rL/Va66F4h57SZMlgBa3+Ziw9HnV66YKcFRV5JPA184IAfvm6jDyxctFc8ruhinZrr0K7rrQA7zUb9BsfseNC9jv4H2KR00OMK7UmXadpcyV6BnOCUsRzBfZbuqxvx7Qz1rihBLR5uGwyv68oLDbXiUAU7Tn7gBtXArqNexm/3WWRP7J6iCYQwMJrQ49d1odQuj0y1cv4m7iYN7DY0T/Xs0MiO2C1GGXnRe1t0GOtm2NTWj8vNP4oHTrGSUfcS5p3wB4UzpaTMhutl2pna9qKCgY6l1JFXtOTAOwldLe6norcoN+hmPvSmQ8pt5dxOEBWuO44SLVMDSlQ43vHhp984iAeRm+cRkTYxXjmfosejyXXynGXQi5FF8TxOyKc7XjKAszcvXS7oEKXa9HLf6RrVyE4rPYzxLqZ0keOoT+U/7Tns1q1cBwT6rVzdOVK7FY0ogbsNb0nq1ahD573JodCddnId7ILsZd6eFSrq1l66UyMf10i9Stx+wTPMR0GxuZ4UCW63W+zgqBUX8ovo+ItUw/u4QxcsNOhb+QYxGsJFj14ZEaPp8xdHsRYfDx2SDl38vJd2o5Uj2Jq2jBSNOzW/GIL2MPFGGntBp3CZqFgnhyfeNnMufjgsU1YTmWnuJdzALRNlBsQ5rNbA0FFCYbt5/EJjLQW83ULuyEyZubZRGDvoDrmnVBO3ea80Qw1s8GT27MDmyvYEwd4Kvy682jLWn0NBKbUxmPMyWDhi4XaPevzUwXWM8kIJ4FVwk9REfU19tVK+eMCtk254ZuxzLLEjPoDf2jRavMOHAkPuPOyhY8AJ0hIuQZAvDxMuaBk6XoaozUfUfs2n/5x5heMrTqL00YzcU+FudINnJiGNXI+5/Cqbk+46po+4FaW84xhdlC41fTTnsrPOXp7pc0PFfKE/7E4OZ4eT7lB39iYAppyeILGpC9yJq6JIe60wfXqDNR0KDNrk7umdQpfu0njB48fX4Y+0c9Ls0nS1D+VVYfrRBgOhOmWtzeYQdM4MTZli/JTJyc3Tk/AXn39f10qzFk+KTPeyfeYflU81cVHKptqi85aY9NiFq+828W+8Gdc9g4N7BFZ29DIdamzV6OhaXX9+nUpp47jPk0fj3gTAZ//Ev/Nf8aKFW0v3HXgo0wH9rEb10hvqUlJjTmHluL0U5PnWX2+iFj77I/yDGycqOxIrtbMMJqaXIehgV+Ph0/R0T94ruM/jfvf9zQUE3Dy/Hf7JTV90TxNO54DBXu0DwI08wHIjw4buuA1+CHhD4Ob5o/AF1jLonk+ZdmlgGE7H0zSwRlApyLtKW4RJseADZH9EvBi4eY6dE3MbZ5rAgukWZKvpQUyN4T2NUkThr+TjzBDG1fkp5CXAzfMn4YtMJkhzEIvmYVCTUfed+l2jLwkzNhIfSLcCvvzJwgIN3DzD7rirdz1KwtVn6FHS5PIzyi1BBSWxj3j/msRzUq1AZkEf8MAf4dcpfyeZESdVAXJHwP8lN/6Y6qTAOoNXPyLmD1rvBf0BVSsyXt7c5ICbZx/id6h4AHTu/fqEvAct0wHJqeTjW1QjAAMA2anpoCfZ/Q32QapH44s/i3gT4ObZ+/zNAU/RQdfrBlUZgzukgT6dXHw/CVigQy/UiwfgBCfpPdgQfR/zUsDN8zvhm0msCI4R9aZVKH1AfSCOZaFpivpQAOjbzxMEnDb6d8JLA0fuOBkx0CKeej8j5NPi1YaN5K5BcZayELCUOOogM0KKlwGO3HE8AiB/oXEgg6oqB3yt8gBDpbC38oTmZYEj57SqecHsVYnv5WoSO0HZ2dylRNE9GSsLCzBws4mNWmQvgJufHzgeLn7CDjkDdSFAiEZGHHGrv0iBo7nivvSKGk9TABBx1mOYBE/ARwVbQHiJzR0xWcf4r19vyls4miuSYSo2W+pt5+BUJvrAOPbJ4FFMvrIUsd8k4RzuOP/ieFPAzTM8VyTBnNjTqRdfwRluZO9acbgIHtOj3k8nbI4R/d4PPG8auHn+ZfhRkkYWzm3VOyjAb/SI8iqduLHho7eVpQj/CxtDnCX7McUrAG6efxp+eCxpqnxVQYq8bzu+CJzuUJ3eIg7nwrdwZ/8pzSsCjuaKvuouwoJPwYsgarFRgidlqkVtYQwethPuNScCXiFw5I7XwfqojgqCU5dk+B8l7JIUrS8tRGyy0OSHmMwPBLxi4CaZK6JpitiIys2WJClHnBGK4Uh8IFsllT0CLnaZyNKROcAbES8AHLljVKDY/shc8bEEgZhe1G+I6ZMlVmTnPorjFTR/w/3rmahDQ8DNs/vhP6FRJM4DyJ4K1UAInYas4uz9EUg840aXx2mB78W8EDA9VxQHHx5YFemXTJFQK/x9kvyamRjIMBxExacdsAI4misOwPkqRKz4Uq3wM3j8+RrAtb5oarYFrDssR0ZW4IBVwNFccQ1c1VgRnTUwUz0rEH4Ku62w9dSHfKbdMWgUp+TKIgeMca/CwJE7nsKxkJ+qivoY2vBj2BWFzg3KWlBaZvfdrIJOb0QyLW2IduG3xevX3lmCiM+wZ9+E3eqIvftTjX0LoXknIQNye3rLsoNVPOAnq7IR4xOHJHLAAe3Pi9evX79y7Z1AYuYodbslW+xy46rofXMH+nRk+NGUOMPWCuWKDLmRb9Izhog2UAgMMJPU7Ui5dJphxxGaWkYxEjLZBr8Pcw37Q94BE9orRBGwkJmkbvuKQymzCMWrsWHbK/ytPbSw/2Qd8NWrv/18JaFlgQXMxB0b3CQ3pdO/a8oDPjOLScle/e13ljYFjJgZu03miua0TUf9y8a/GTJxSEJaETDHTNyxMa0xweSmYeBoRgjRAsAMM5krmtKQyekMinwrk0CRu/39upgWBkbM0YA2uutzzCKa/XrXXxgHlBU4MmLRXNGMBqyP2zYJ/KuSVgkcMpPUrRkN2ChmZHDX/7P/KWl1gAPtvG+uVqa3CVN69cWtxypcHeAH3/zHtG8qSydPH98qCvzgwV+f2ObIovf+omhmKfCLb/78yDZBZr1+KUWWAD/45m+Go4456eT5FZgZAn7x4J1/lPQ192XpCRUiwQZMDMwaqpOKjOKnL6mKPvnTLaEBEwGzhurJ/dvzr3s+3bv13SvqT6EBSwFzhurRt/+9M+9659fz3d2H96ie/UnagHHArKE6uXP3dOfL+dc7v97bXXy4+/KT5IWTe5wBo4GRoaJuz+v7O6dL1eINiQPkr96jXmIN2DWqcf9OR1SPvt65u7S0Uzk//CogDpgfPqWajjZgETBrqE5un54GuEs7VQkqKX0SEgfN/DnVsxMDdo0YKrrnfvph0JeXKsobEwfIHz+nX8YG7FoqovryXdSXq8ubECPmz+h+e+/xrccvHrygDdWT26RxEW81A8saTRwgM6751ReMofr02wS3wrxBuyXEi4u7i8+Febn2nbtRXw55KxJPisUQcz0bK3S6lE4rzRuM14eLLDLjmonTTXS36rwpYuSao6AzcroU712zyWgrOlnkhV3z6z/Yvox4ly4BLyLmGxn1bK4vXyLegPjjFPHiIk8b8L5bsUwHrPZXaeI079eXhjdQmjjNa7uOZvUdT3zJedPELO/pt7brZ14c8aXnrdU+fwgBn35ou27liCGmeU0ui14ovXwoAj69b7te5Ykifit4a7XPdnngnUvNSxFHvJVZTsmrp7s08OXnrdXu7SbAbwNvRBzyVmi5rIie72Lgqi0f5RdaeHqbeEPiCi6XFdGr3beLNyC2tXz0fxROpvg0dQm5AAAAAElFTkSuQmCC"
               alt="Penguin Logo">
          <figcaption>Penguin Twitter</figcaption>
      </figure>
    <h1>Welcome to Fake Twitter</h1>
  </header>

  <nav>
    <ul>
      <li>Home</li>
      <li>Post a Tweet</li>
      <li>Search</li>
      <li>About</li>
    </ul>
  </nav>

  <main>
    <section>

      <ul>
        <li>Tweets</li>
        <li>Following</li>
        <li>Followers</li>
      </ul>

      <article>
          <h2>Aaron Dennis</h2>
          <h4>@AMDennis87</h4>
          <h5>Portland, Oregon</h5>
      </article>

      <article>
          <figure>
            <img src="https://image.flaticon.com/icons/png/128/118/118781.png">
          </figure>
          <h3>Aaron</h3>
          <h5>@AAron</h5>
          <p>Tweet Tweet</p>
      </article>

      <article>
        <figure>
          <img src="https://image.flaticon.com/icons/png/128/118/118781.png">
        </figure>
        <h3>Aaron</h3>
        <h5>@AAron</h5>
        <p>Tweet Tweet</p>
    </article>

    <article>
      <figure>
        <img src="https://image.flaticon.com/icons/png/128/118/118781.png">
      </figure>
      <h3>Aaron</h3>
      <h5>@AAron</h5>
      <p>Tweet Tweet</p>
  </article>

    </section>

    <aside>
      <h3>People You Should follow</h3>
      <p>John @TheJohn</p>
      <p>Jim @JimBim</p>
      <p>Tina @TinaTina</p>
    </aside>

  </main>

  <footer>
    <p>Penguin Twitter &copy; 2019</p>
    <ul>
      <li>Careers</li>
      <li>Notifications</li>
      <li>Tweet</li>
    </ul>
  </footer>

</body>
</html>