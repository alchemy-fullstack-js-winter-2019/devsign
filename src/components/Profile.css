.profileHeader {
  margin: 0;
  text-align: center;
  display: inline;
	position: relative;
	font: 2em Helvetica, Sans-Serif;
	letter-spacing: -2px;
  color: rgba(255,0,0,0.5)
}

.tweetWrapper {
  text-align: center;

  img {
    border-radius: 0;
  }
}