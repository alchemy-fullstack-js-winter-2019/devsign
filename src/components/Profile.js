import React from 'react';
import UserDetails from './UserDetails';
import CreateTweetForm from './CreateTweetForm';
import Nav from './Nav';
import Tweets from './Tweets';
import { Main, Section } from '../styles/mainStyle';

const tweets = [
  {
    profileImage: 'http://capepremierrealty.com/wp-content/uploads/2018/11/generic-profile-icon-7.jpg.png',
    name: 'Kaiya',
    handle: '@ladybeard',
    text: 'tweet tweet tweet'
  },
  {
    profileImage: 'http://capepremierrealty.com/wp-content/uploads/2018/11/generic-profile-icon-7.jpg.png',
    name: 'Kaiya',
    handle: '@ladybeard',
    text: 'tweet tweet tweet'
  },
  {
    profileImage: 'http://capepremierrealty.com/wp-content/uploads/2018/11/generic-profile-icon-7.jpg.png',
    name: 'Kaiya',
    handle: '@ladybeard',
    text: 'tweet tweet tweet'
  },
  {
    profileImage: 'http://capepremierrealty.com/wp-content/uploads/2018/11/generic-profile-icon-7.jpg.png',
    name: 'Kaiya',
    handle: '@ladybeard',
    text: 'tweet tweet tweet'
  },
  {
    profileImage: 'http://capepremierrealty.com/wp-content/uploads/2018/11/generic-profile-icon-7.jpg.png',
    name: 'Kaiya',
    handle: '@ladybeard',
    text: 'tweet tweet tweet'
  },
  {
    profileImage: 'http://capepremierrealty.com/wp-content/uploads/2018/11/generic-profile-icon-7.jpg.png',
    name: 'Kaiya',
    handle: '@ladybeard',
    text: 'tweet tweet tweet'
  } 
];

const  user = {
  name: 'Kaiya',
  handle: '@ladybeard',
  profileImage: 'http://capepremierrealty.com/wp-content/uploads/2018/11/generic-profile-icon-7.jpg.png',
  bannerImage: 'https://upload.wikimedia.org/wikipedia/commons/b/bc/Saddle_tor_to_Hey_tor_pano.jpg',
  bio: 'Fullstack MERN Developer / Animal Lover / Hiker',
  location: 'Portland, OR'
};

function Profile() {
  const { name, handle, profileImage, bio, location } = user;
  return (
    <>
    <Nav />
      <CreateTweetForm />
    <Main>
      <UserDetails
        name={name}
        handle={handle}
        profileImage={profileImage}
        bio={bio}
        location={location}/>
      <Section>
        <Tweets tweets={tweets}/>
      </Section>
    </Main>
    </>
  );
}

export default Profile;
