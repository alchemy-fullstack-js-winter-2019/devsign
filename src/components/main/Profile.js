import React from 'react';
import styles from 'styled-components';
import profilePic from '../../../assets/drew.jpg';

const ProfileAvatar = styles.img`
@media (min-width: 374px) {
  border-radius: 50%;
  width: 15vw;
  height: 8vh;
  border: 3px solid #b7fbff;
  }
@media (min-width: 700px) {
  width: 8vw;
  height: 8vh;
  }
@media (min-width: 700px) {
  width: 6vw;
  height: 8vh;
  }
`;

const ProfileUser = styles.aside`
@media (min-width: 374px) {
  font-family: 'Muli', sans-serif;
  color: #ffa1ac;
  margin: 10px;
  font-size: 25px;

  }
`;


function Profile() {
  return (
   <ProfileUser>
    <section>
      <h2><i className="fas fa-kiwi-bird"></i> PERSONAL USERNAME </h2>
      <ProfileAvatar src={profilePic} width="10%" />
      <p> A person who loves you will never kick you when your down or pour salad on your wounds</p>
    </section>
  </ProfileUser>
  );
}

export default Profile;