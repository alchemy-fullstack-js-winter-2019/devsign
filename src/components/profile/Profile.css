.Home {
    display: grid;
    padding: 1em;
    figure {
      width: 100%;
      margin: auto;
    }
  
    #banner {
      
      width: 100%;
      margin: auto;
      display: block;
    }
    #about {
      margin-top: 1em;
      margin-bottom: 1em;
      display: grid;
      grid-template-columns: 50% 50%;
      img {
        width: 50%;
      margin: auto;
      }
      p {
        display: inline;
      }
    }
    #tweets {
      margin: -1em;
    }
  }