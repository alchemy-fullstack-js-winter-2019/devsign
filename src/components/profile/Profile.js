import React from 'react';

function Profile() {
  return (
    <>
      <img src="https://via.placeholder.com/150" />
    </>
  );
}

export default Profile;
