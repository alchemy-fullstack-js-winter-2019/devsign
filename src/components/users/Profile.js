import React, { Fragment } from 'react';
import Header from '../app/Header';
import Footer from '../app/Footer';
import Chirp from '../chirps/Chirp';
import styled from 'styled-components';
import 'normalize.css';

const Main = styled.main`
  .background {
    height: 100px;
    z-index: -1;
    width: 100vw;
    position: absolute;
  }
  .prof {
    height: 100px;
    margin-top: 50px;
  }
  th {
    font-weight: 200;
  }
  table {    
    text-align: center;
  }
  .section {
    display: flex;
    align-items: flex-end;
    justify-content: space-around;
  }
  h3 {
    font-weight: 300;
    margin-bottom: 0;
    margin-top: 0;
    font-size: 0.9em;
  }
  .info {
    display: flex;
  }
  .name {
    text-align: center;
    margin-top: 0;
    margin-left: 10px;
    margin-right: 10px;
  }
  .description {
    margin-left: 5px;
    margin-top: 10px;
    margin-right: 10px;
    font-size: 0.8em;
    text-align: center;
  }
  p {
    padding: 0;
    margin: 0;
    font-size: 0.8em;
  }
  h2 {
    font-weight: 300;
    text-align: center;
    margin-top: 5px;
  }
  ul {
    padding: 0;
  }
  li {
    list-style-type: none;
    border: 1px solid lightgray;
    margin: 15px 15px;
    padding: 10px;
  }
 
  @media only screen and (max-width:600px)  { 
    .trending {
      display: none;
    }
  }
`;
export default function DesktopProfile() {
  return (
    <Fragment>
      <Header />
      <Main>
        <section>
          <img src="../assets/background.jpg" alt="background" className="background"/>
          <div className="section">
            <img src="../assets/prof-pic.png" alt="prof pic" className="prof"/>
            <table>
              <thead>
                <tr>
                  <th>Chirps</th>
                  <th>Followers</th>
                  <th>Following</th>
                </tr>
              </thead>
              <tbody>
                <tr>
                  <td>593</td>
                  <td>388</td>
                  <td>301</td>
                </tr>
              </tbody>
            </table>
          </div>
        </section>

        <section className="info">
          <div className="name">
            <h3>Kristin Hortsch</h3>
            <p>@kristinhortsch</p>
          </div>
          <div className="description">
            <p>Lover of the laughing, sports, and all things coding.</p>
          </div>
        </section>

        <hr></hr>

        <section>
          <h2>Tweets</h2>
          <ul>
            <li><Chirp /></li>
            <li><Chirp /></li>
            <li><Chirp /></li>
            <li><Chirp /></li>
            <li><Chirp /></li>
          </ul>
        </section>

        <aside className="trending">
          <h3>Trending</h3> 
          <ul>
            <li>#Trending 1</li>
            <li>#Trending 2</li>
            <li>#Trending 3</li>
            <li>#Trending 4</li>
            <li>#Trending 5</li>
          </ul>
        </aside>
      </Main>

      <Footer />
    </Fragment>
  );
}
