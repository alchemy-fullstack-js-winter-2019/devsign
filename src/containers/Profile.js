import { connect } from 'react-redux';
import Profile from '../components/profile/Profile';
import { getUserTweetsAction } from '../actions/Tweets';
import { getUserTweets } from '../selectors/Tweets';



const mapStateToProps = state => ({
    tweets: getUserTweets(state)
});

const mapDispatchToProps = dispatch => ({
    fetchTweets() {
        dispatch(getUserTweetsAction());
    }
});

export default connect(
    mapStateToProps,
    mapDispatchToProps
)(Profile);
