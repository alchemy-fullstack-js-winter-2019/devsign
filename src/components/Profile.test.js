import React from 'react';
import Profile from './Profile';
import renderer from 'react-test-renderer';

describe('Profile component', () => {
  it('matches a snapshot', () => {
    const tree = renderer.create(
      <Profile />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
