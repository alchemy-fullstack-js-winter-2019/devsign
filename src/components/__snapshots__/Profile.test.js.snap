// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login matches a snapshot 1`] = `
<main>
  <article>
    <figure>
      <img
        alt="user avatar"
        src="https://pbs.twimg.com/media/C8QsNInXUAAyjZQ.jpg"
      />
    </figure>
    <h3>
      Teonna
    </h3>
    <h4>
      T_on_A
    </h4>
    <p>
      Bio about me
    </p>
  </article>
  <article>
    <ol>
      <li>
        <p>
          January 23, 2019
        </p>
        <p>
          MongoDB for lyfe #MONGODB
        </p>
      </li>
    </ol>
  </article>
  <aside>
    <p>
      Trends
    </p>
  </aside>
</main>
`;

exports[`Profile matches a snapshot 1`] = `
<main>
  <article>
    <figure>
      <img
        alt="user avatar"
        src="https://pbs.twimg.com/media/C8QsNInXUAAyjZQ.jpg"
      />
    </figure>
    <h3>
      Teonna
    </h3>
    <h4>
      T_on_A
    </h4>
    <p>
      Bio about me
    </p>
  </article>
  <article>
    <ol>
      <li>
        <p>
          January 23, 2019
        </p>
        <p>
          MongoDB for lyfe #MONGODB
        </p>
      </li>
    </ol>
  </article>
  <aside>
    <p>
      Trends
    </p>
  </aside>
</main>
`;
